### PR TITLE
Reset timer in depletion for transport performance

### PR DESCRIPTION
**Pull request by rockfool**
This PR is to indeed reset the timer to display the transport speed when running depletion. This bug was found by a user who tried to tally the "calculation rate". This bug will lead to the dramatical decrease of the nominal "value" of calculation rate in the depletion.  I fix it by adding a reset_timers function in C API and apply it in `depletion.operator`.

### CODE REVIEW

File: src/timer.cpp [hunk 1]
```diff
@@ -87,4 +87,9 @@ void reset_timers()
   simulation::time_event_death.reset();
 }
 
+void restart_timers()
+{
+  simulation::time_total.start();
```
Comment (paulromano) [inline src/timer.cpp:92]: I think a better solution than adding a `restart_timers` function is to just have `openmc_init` and `openmc_run` start _and_ stop the `time_total` timer.

File: openmc/lib/core.py [hunk 2]
```diff
@@ -303,10 +307,16 @@ def plot_geometry():
 
 
 def reset():
-    """Reset tallies and timers."""
+    """Reset tallies but keep timers."""
```
Comment (paulromano) [inline openmc/lib/core.py:310]:
```suggestion
    """Reset tally results."""
```

File: src/timer.cpp (hunk 1 above)
Comment (rockfool) [reply src/timer.cpp:92]: Thanks for your comments. However, depletion calls `openmc_init` only once (`init`->`run`->`reset`->`run`->...).  If I start and stop `time_total` timer in `openmc_init` and `openmc_run`, the timer only counts the total time for the first batch of neutron transport. (`start`->`stop`->`reset`->`stop`->`reset`->...) All zeros for other batches.

**Comment by paulromano** [comment]
Thanks @rockfool!

**Pull request merged**